### ISSUE TITLE
Support created collection / partition time in milvus proto api.

### ISSUE THREAD

**Issue by nameczz** [milvus-io/milvus#6343]
**Please state your issue using the following template and, most importantly, in English.**

**Is your feature request related to a problem? Please describe.**
Support created collection / partition time in milvus proto api.

**Describe the solution you'd like**
May be support created collection / partition time in showcollections api

**Describe alternatives you've considered**

**Additional context**
Add any other context, code examples, or references to existing implementations about the feature request here.

**Comment by shanghaikid** [comment]
User would like to view a collection/partition list sorted by created time in [milvus-insight](https://github.com/milvus-io/milvus-insight), there is no such api or proto can do this.  Am I correct? @czs007

**Comment by DragonDriver** [comment]
> User would like to view a collection/partition list sorted by created time in [milvus-insight](https://github.com/milvus-io/milvus-insight), there is no such api or proto can do this. Am I correct? @czs007

Yeah, there is no such api now. We can provide this but maybe the timestamp is not the real time. Milvus uses a Timestmap Oracle (tso) to assign timestamp to request, the timestamp assigned by tso is globally increasing, so you can sort collection/partition list by this timestmap. The timestamp is a hybrid time which contains two part: physical part and logical part, the physical part can be easily calculated by the timestamp, so should we return the timestamp or the physical part?

**Comment by DragonDriver** [comment]
> > User would like to view a collection/partition list sorted by created time in [milvus-insight](https://github.com/milvus-io/milvus-insight), there is no such api or proto can do this. Am I correct? @czs007
> 
> Yeah, there is no such api now. We can provide this but maybe the timestamp is not the real time. Milvus uses a Timestmap Oracle (tso) to assign timestamp to request, the timestamp assigned by tso is globally increasing, so you can sort collection/partition list by this timestmap. The timestamp is a hybrid time which contains two part: physical part and logical part, the physical part can be easily calculated by the timestamp, so should we return the timestamp or the physical part?

Maybe we can return both the timestamp and the physical part. The timestamp is used to sort and the physical part is used to be shown? @czs007

**Comment by shanghaikid** [comment]
can you give us an example about Timestmap Oracle (tso)?

**Comment by DragonDriver** [comment]
For example, `426152581543231492` is a timestamp assigned by tso. The last 18 bit of timetamp is of logical part, the front 46 bit is of physical part.

Given a ts, we can calculate the physical part and logical part:

```go
const (
	logicalBits     = 18
	logicalBitsMask = (1 << logicalBits) - 1
)

// ParseTS parses the ts to (physical,logical).
func ParseTS(ts uint64) (time.Time, uint64) {
	logical := ts & logicalBitsMask
	physical := ts >> logicalBits
	physicalTime := time.Unix(int64(physical/1000), int64(physical)%1000*time.Millisecond.Nanoseconds())
	return physicalTime, logical
}
```

**Comment by nameczz** [comment]
I think we only need physical part time, so should we calculate the physical part in Milvus?

**Comment by shanghaikid** [comment]
any update for this?

**Comment by czs007** [comment]
> I think we only need physical part-time, so should we calculate the physical part in Milvus?

I think it is better to let the SDK calculate the physical timestamp based on the hybrid logic timestamp, which can save meta storage space and network transmission.

**Comment by DragonDriver** [comment]
> > I think we only need physical part-time, so should we calculate the physical part in Milvus?
> 
> I think it is better to let the SDK calculate the physical timestamp based on the hybrid logic timestamp, which can save meta storage space and network transmission.
> @nameczz

I don't think so. If milvus doesn't do this, then every sdk will have an implementation of parsing hybrid timestamp. I think the simpler sdk is, the less duplicated work wil be produced. And though we won't change the rule of parsing hybrid ts, i don't think it's reliable to do this in sdk.

**Comment by DragonDriver** [comment]
> any update for this? @DragonDriver

already resolved. pls feel free to try this new feature.
/assign @nameczz 
/assign @shanghaikid

**Comment by xiaofan-luan** [comment]
closed since it is already solved  #6700
